src/client/client_conduit.js

```diff
@@ -45,6 +48,12 @@ export default class ClientConduit extends Conduit {
       this.game.app.renderer.render(this.sightCircle, this.game.sightRangeTexture, false, null, false);
     }
 
+    if (this.health !== Constants.CONDUIT_HEALTH && this.health !== 1) {
```

Comment:
Checking `this.health !== 1` seems potentially confusing, can it be replaced with `this.team !== NEUTRAL`?